lib/galaxy/datatypes/proteomics.py

```diff
@@ -134,6 +134,12 @@ class MzML(ProteomicsXml):
     blurb = 'mzML Mass Spectrometry data'
     root = "(mzML|indexedmzML)"
 
+class NmrML(ProteomicsXml):
```

Comment:
If you want to sniff this you need to add the datatype the the sniffers section in datatypes_conf.xml.sample